### ISSUE TITLE
Draggable table should be accessible by keyboard and screen reader

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#5936]
**Describe the issue. What is the expected and unexpected behavior?**
The Composable Draggable row table should be accessible by keyboard and by screen reader. Currently it works only by mouse. I imagine that this would follow similar accessibility considerations as the [draggable datalist component](https://www.patternfly.org/v4/components/data-list/#draggable) that we implemented with this [PR](https://github.com/patternfly/patternfly-react/pull/4852). 

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
Try to use a keyboard or screen reader to interact with[ this example](https://patternfly-react-add_draggable_table.surge.sh/components/table/#composable-draggable-row-table). 

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug
